github/commitstatus.go

```diff
@@ -31,12 +31,14 @@ func (e *StatusError) Error() string {
 	return fmt.Sprintf("%s\n%s", e.What, e.Details)
 }
 
+// Default GitHub hostname
+const GhDefaultHostname = "github.com"
+
 // API is the GitHub API endpoint.
```

Comment:
// API is the default GitHub API hostname.

Reply:
Fixed in: https://github.com/Pix4D/cogito/commit/66f1cf52187b4780f4fb0775009bab43f2a2155d

Reply:
Actually, is this correct?

` API is the default GitHub API hostname`? hostname? actually in this case its not hostname. It contains a protocol part (`https://`) and the API hostname part: `api.github.com`, so more correct would be base url?

Reply:
Fixed.